### PR TITLE
Fixes #9

### LINKED ISSUE #9: call user-post-details should be copied from the old docs
When copying a call, make sure:
- The content is generic and not specific to a use case, it only describes the call independently
- You use the correct sections (see #2)
- You `.. highlight:: js` at the start of the document
- You use existing request field descriptions for:
 - `user`
 - `partner-identifier`

### PR DESCRIPTION
**Pull request by herrreiprich**

### CODE REVIEW

File: docs/calls/user_post_details.rst [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+        depending on the identifier-type (string).
+    token (optional)
+        A token can be used to authenticate the user (string).
+
+        For example: if the identifier type is username and the identifier is the user's username,
+        then token is used for authentication instead of a password.
+
+details
+    first-name
+        A string. Can be null.
```
Comment (apheleia) [inline docs/calls/user_post_details.rst]: Please do all types like in other calls: either only the type, or an explanation and the type in brackets.

Examples:
```
first-name
    string
```
```
first-name
    The first name of the user (string).

File: docs/calls/user_post_details.rst [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    }
+
+Response::
+
+    [
+        {
+            "user-post-details": {
+                "success": true
+            }
+        },
```
Comment (apheleia) [inline docs/calls/user_post_details.rst]: Please separate the two possible responses:

```
{
    "user-post-details": {
        "success": true
    }
}
```
```
{
    "user-post-details": {
        "success": false,
        "reason": "Could not validate VAT number: DE1234567"
    }
}
```

File: docs/calls/user_post_details.rst [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+reason
+    An optional response field. If present, it is a string stating the reason for an error.
+
+Status codes
+~~~~~~~~~~~~
+200 OK
+   Request was processed successfully
+401 Unauthorized
+   The token, username or identifier type were incorrect
+
```
Comment (apheleia) [inline docs/calls/user_post_details.rst]: Please add:
`400` if the country or the VAT are invalid (refer to `reason` field in response)

File: docs/calls/user_post_details.rst [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+        The identifier is something that uniquely identifies the customer,
+        depending on the identifier-type (string).
+
+    token (optional)
+        A token can be used to authenticate the user (string).
+
+        For example: if the identifier type is username and the identifier is the user's username,
+        then token is used for authentication instead of a password.
+
+    first-name
```
Comment (apheleia) [inline docs/calls/user_post_details.rst]: the details are in a separate object `details`, see example request

File: docs/calls/user_post_details.rst [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+    }
+
+Response::
+
+    {
+        "user-post-details": {
+            "success": true
+        }
+    }
+
```
Comment (apheleia) [inline docs/calls/user_post_details.rst]: please split. the two responses should not be listed as one

Comment (herrreiprich) [inline docs/calls/user_post_details.rst]: how should I split the response. The objects are separated. Do you want a headline for the two responses here?

Comment (herrreiprich) [inline docs/calls/user_post_details.rst]: got it.

Comment (apheleia) [inline docs/calls/user_post_details.rst]: Just so it ends up in two separate "code boxes". The HTML should be rendered as separate `div`s.

**Pull request merged**